satpy/readers/seviri_base.py

```diff
@@ -353,6 +353,81 @@
                          'ALPHA': 0.9981,
                          'BETA': 0.5635}}
 
+# Calibration coefficients from Meirink, J.F., R.A. Roebeling and P. Stammes, 2013:
+# Inter-calibration of polar imager solar channels using SEVIRI, Atm. Meas. Tech., 6,
+# 2495-2508, doi:10.5194/amt-6-2495-2013
+#
+# The coefficients are stored in pairs of A, B (see function `get_meirink_slope`) where the
+# units of A are µW m-2 sr-1 (cm-1)-1 and those of B are µW m-2 sr-1 (cm-1)-1 (86400 s)-1
+#
+# To obtain the slope for the calibration, one should use the routine get_seviri_meirink_slope
+
+# Epoch for the MEIRINK re-calibration
+DATE_2000 = datetime(2000, 1, 1)
+
+MEIRINK_COEFS = {}
+MEIRINK_COEFS['2013'] = {}
+
+# Meteosat-8
+
+MEIRINK_COEFS['2013'][321] = {'VIS006': (24.346, 0.3739),
+                              'VIS008': (30.989, 0.3111),
+                              'IR_016': (22.869, 0.0065)
+                              }
+
+# Meteosat-9
+
+MEIRINK_COEFS['2013'][322] = {'VIS006': (21.026, 0.2556),
+                              'VIS008': (26.875, 0.1835),
+                              'IR_016': (21.394, 0.0498)
+                              }
+
+# Meteosat-10
+
+MEIRINK_COEFS['2013'][323] = {'VIS006': (19.829, 0.5856),
+                              'VIS008': (25.284, 0.6787),
+                              'IR_016': (23.066, -0.0286)
+                              }
+
+# Meteosat-11
+
+MEIRINK_COEFS['2013'][324] = {'VIS006': (20.515, 0.3600),
+                              'VIS008': (25.803, 0.4844),
+                              'IR_016': (22.354, -0.0187)
+                              }
+
+
+def get_meirink_slope(meirink_coefs, acquisition_time):
+    """Compute the slope for the visible channel calibration according to Meirink 2013.
+
+    S = A + B * 1.e-3* Day
+
+    S is here in µW m-2 sr-1 (cm-1)-1
+
+    EUMETSAT calibration is given in mW m-2 sr-1 (cm-1)-1, so an extra factor of 1/1000 must
+    be applied.
+    """
+    A = meirink_coefs[0]
+    B = meirink_coefs[1]
+    delta_t = (acquisition_time - DATE_2000).total_seconds()
+    S = A + B * delta_t / (3600*24) / 1000.
+    return S/1000
+
+
+class MeirinkCalibrationHandler:
+    """Re-calibration of the SEVIRI visible channels slop (see Meirink 2013)."""
+
+    def __init__(self, coefs=MEIRINK_COEFS, calib_mode=None):
+        """Initialize the calibration handler."""
+        if calib_mode is None:
+            raise ValueError("Missing calib_mode")
```

Comment:
Is there a use case where `calib_mode` is actually `None`?

Reply:
I could remove the `=None` default to `calib_mode` but that would make it a non-keyword argument after a keyword argument. I have no good idea here :-/